### DISCUSSION TITLE
Welcome to pictode Discussions!

### DISCUSSION THREAD

**Discussion by JessYan0913** [JessYan0913/pictode#5] [Announcements]
## 👋 Welcome! 欢迎！

  We’re using Discussions as a place to connect with other members of our community. 我们使用 Discussions 作为与社区其他成员互相交流的地方。
  
  We hope that you: 我们希望你能够：
  * Ask questions you’re wondering about. 提出你感兴趣的问题。
  * Share ideas. 分享你的想法。
  * Engage with other community members. 参与与其他社区成员的互动。
  * Welcome others and are open-minded. 让其他人感到受欢迎，保持开放心态。Remember that this is a community we
  build together 💪. 请记住，这是我们共同建立的社区 💪。

  To get started, comment below with an introduction of yourself and tell us about what you do with this community. 为了开始，您可以在下方发表一篇自我介绍，告诉我们你在这个社区中做什么。

**Comment by php-chen** [comment]
会出微信小程序版本吗

**Comment by JavanFeng** [comment]
多图层版本预计什么时候完成初版？

**Comment by php-chen** [comment]
劳 
http://www.yh6dvs.cn ，来这里找喜我，注册后把ID发给我，我们试试能不能用视频送看牌作弊。
